README.md

```diff
@@ -1,5 +1,20 @@
 # Firefly
 
+## Intro
+
+Firefly is an open-source web-based UI library for astronomical data archive access and visualization developed in Infrared Processing and Analysis Center (IPAC) at [Caltech](https://caltech.edu).
+The development started with the Spitzer Heritage Archive (SHA), and continued in the WISE Image Archive, Planck Image Archive, and other web applications in the NASA/IPAC Infrared Science Archive (IRSA) applications. It was open sourced in 2015, hosted at GitHub.
```

Comment:
"... and other web applications serving data from the NASA/IPAC Infrared Science Archive (IRSA).", perhaps?  (Trying to avoid repeating "applications".)